code/modules/mining/equipment/mining_tools.dm

```diff
@@ -9,6 +9,7 @@
 	throwforce = 10
 	inhand_icon_state = "pickaxe"
 	worn_icon_state = "pickaxe"
+	worn_icon = 'icons/mob/clothing/belt.dmi'

```

Comment:
wrong. already handled by build_worn_icon. this will result in the wrong icon being used when on a back or belt.

Reply:
Reverted. Had the worn_icon set to make the belt_mirror use a working icon. I'll add it to the list of missing textures.